src/Sulu/Bundle/ContentBundle/Form/Type/BasePageDocumentType.php

```diff
@@ -62,8 +61,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $builder->add('shadowLocale', TextType::class); // TODO: Should be choice of available shadow locales
         $builder->add(
             'authored',
-            DateType::class,
-            ['widget' => 'single_text', 'model_timezone' => 'UTC', 'view_timezone' => 'UTC']
```

Comment:
@Edvinas9 why is this needed? the problem with that is - that the type of the value in the database will change with this line. If this is needed - how could we ensure that the type in the database is DateTime

Reply:
I've simply added type="DateTime" which seems to serialise DateTime to a suitable format (`2017-05-01T06:14:00+0000`).

Is anything wrong with that?

Reply:
I was struggling to get `DateTimeType` to accept datetime string values (for example: `2017-05-01T06:14:00+0000`) so temporary put TextType in place just because I had to get this working ASAP for a client's project. I've created a PR early just to get some feedback and/or suggestions on this.

However, I gave it another go and seems like `'widget' => 'single_text'` option works just fine. Are you happy with that?